.github/workflows/main.yaml

```diff
@@ -54,3 +54,26 @@ jobs:
         run: |
           cd docker
           bash build_images.sh
+  build_and_deploy_docs:
+
+    # The type of runner that the job will run on Ubuntu 18.04 (latest)
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be
+    # executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE,
+      # so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASS }}
+
+      - name: Build and push docker
+        run: |
+          cd docs/docker
+          bash build.sh ${GITHUB_REF##*/}
```

Comment:
this will build and tag based on the name of the branch. We should tag based on the version, which can be taken from the __init__.py file.

Reply:
If we use the same version for the release and these PRs won't the docker images get clobbered ? Or how do we pick the one intended for public consumption ?